README.md

```diff
@@ -231,6 +237,51 @@ By default, `beekeeper-scheduler-apiary` listens on port 8080, `beekeeper-path-c
 | `scheduler-delay-ms`                | No       | Amount of time (in milliseconds) between consecutive cleanups. Default value is `300000` (5 minutes after the previous cleanup completes). |
 | `Metastore-uri`                     | Yes      | URI of the Hive Metastore where tables to be cleaned-up are located. |
 
+## Beekeeper-API
+
+Beekeeper also has an API which provides access to the Beekeeper database and allows to see what metadata and paths are currently being managed.
+
+This allows the user to manually enter a database and a table name and check whether this table has been successfully registered in Beekeeper along with the TTL configuration: the current status of the table, the date and time it will be deleted, the current cleanup delay... etc.
+
+It currently supports two endpoints; one for the expired metadata and another one for the unreferenced paths.
+
+It also supports different filters (see [filtering section](#filtering)).
+
+### Expired metadata endpoint (`GET /metadata`)
+
+This endpoint will return the TTL configuration of all expired partitions that are going to be deleted (or have been deleted) in a specific table. If it is unpartitioned it will just show one object; the table.
+
+It is available in this url; 
+
+    http://beekeeper-api.<address>/api/v1/database/{databaseName}/table/{tableName}/metadata
```

Comment:
whats the address part here?

Reply:
it's `egdl-eks-us-east-1.egdp-prod.aws.away.black`; I wanted to put it in but it's open source and I don't know if that's giving private information? does that make sense?